src/main/java/com/theironyard/services/GuildWarsApiService.java

```diff
@@ -0,0 +1,97 @@
+package com.theironyard.services;
+
+import com.theironyard.entities.Item;
+import com.theironyard.entities.Price;
+import com.theironyard.entities.Recipe;
+import com.theironyard.repositories.ItemRepository;
+import com.theironyard.repositories.RecipeRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestTemplate;
+
+import java.util.*;
+
+@Service
+public class GuildWarsApiService {
+    public static final String RECIPES_URL = "https://api.guildwars2.com/v2/recipes/";
```

Comment:
I would recommend 1 constant for https://api.guildwars2.com/v2/ and then the rest be BASE_URL + "/recipes/".